src/components/ConfirmContent.js

```diff
@@ -111,7 +117,27 @@ function ConfirmContent(props) {
                 {_.isString(props.prompt) ? <Text style={[...props.promptStyles, isCentered ? styles.textAlignCenter : {}]}>{props.prompt}</Text> : props.prompt}
             </View>
 
-            {isCentered ? (
+            {props.shouldStackButtons ? (
```

Comment:
This block is just swapping the buttons since props.shouldStackButtons is default behavior

Reply:
Good shout! Since this PR is urgent, I think we should ignore non-blocking optimizations for now though

Reply:
Gotcha!

Reply:
Could we make an issue to track this change otherwise we never gonna update it 😂